package.json

```diff
@@ -237,6 +244,24 @@
                     "command": "extension.helmDepUp",
                     "group": "2_helm@99"
                 },
+                {
+                    "title": "Create Kubernetes resource",
+                    "when": "resourceLangId == yaml && config.vs-kubernetes.resource-context-menu",
```

Comment:
Need to update the config entry name here and in other `when` clauses.

Reply:
done. (I actually did all the others, but missed this one...)